www/js/modules/main/directives/histogram/histogramChart.js

```diff
@@ -78,113 +64,215 @@ let resize = function() {
         .attr('d', overlayLine);
 
 };
+*/
+/*
+let getSvgWidth = (elm) => {
+    return parseInt(select(elm).style('width'), 10);
+};
+
+let getBarContainerWidth = (svgWidth, clipWidth) => {
+    return svgWidth - clipWidth;
+};
+
+let getElementHeight = (elm) => {
+    return parseInt(select(elm).node().getBBox().height, 10);
+};
+
+let getElementWidth = (elm) => {
+    return parseInt(select(elm).node().getBBox().width, 10);
+};
 
-export default ($log) => {
+let removeElement = (elm) => {
+    select(elm).remove();
+};
+
+let tooltipAdd = () => {
+    select('body').append('div')
+            .attr('class', 'tooltip');
+};
+
+let tooltipShow = (top, left, innerHtml) => {
+    select('.tooltip')
+            .style('visibility', 'visible')
+            .style('top', top)
+            .style('left', left)
+            .html(innerHtml);
+};
+
+let tooltipHide = () => {
+    select('.tooltip')
+            .style('visibility', 'hidden');
+};
+*/
+
+/*
+let getLeadTime = (data) => {
+    return data.map((d) => { return d.leadtime; });
+};
+
+let update = (data, element) => {
+
+    // update x scaleBand
+    x.domain(getLeadTime(data));
+
+    // Bind data
+//    bars.data(data).enter();
+    
+    // Enter
+     bars.selectAll('rect')
+        .attr('height', (d) => { return d.frequency * barHeight; })
+        .attr('x', (d) => { return x(d.leadtime); }) 
+        .attr('y', (d) => { return barContainerHeight - (d.frequency * barHeight) - .5; });
+};
+
+let setup = (data) => {
+    x = setupXscaleBand(data);
+};
+
+let setupXscaleBand = (data) => {
+    let xScaleBand, padding = .62;
+    // setup the x scaleBand for the histogram bars
+    // domain is set in the update function
+    xScaleBand = scaleBand()
+//            .domain(getLeadTime(data))
+            .rangeRound([0, barContainerWidth]);
+    xScaleBand.padding(padding);
+    return xScaleBand;
+
+}
+*/
+let histogramController = function() {};
+
+export default ($log, histogramService) => {
```

Comment:
As you probably see this is the only place that I'm referencing histogramService.